src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -340,13 +339,22 @@ func (cfg Config) cloudPollSleepTime() time.Duration {
 	return time.Duration((CloudConfigPollInterval.Nanoseconds() / 2) + rand.Int63n(CloudConfigPollInterval.Nanoseconds()))
 }
 
-func (cfg Config) fetchCloudConfig() ([]byte, error) {
-	url := cfg.CloudConfig
+// updateFrom creates a new Config by 'merging' the given yaml into this Config.
+// The masquerade sets, the collections of servers, and the trusted CAs in the
+// update yaml  completely replace the ones in the original Config.
+func (updated *Config) updateFrom(newCfg *Config) error {
```

Comment:
Cloud config override anything locally. Those not in cloud config will remain unchanged. Removed lots of extra stuff.